effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -221,6 +221,8 @@ class RecursiveDescent(positions: Positions, tokens: Seq[Token], source: Source)
      case Var(id) =>
        val tgt = IdTarget(id)
        Return(Call(tgt, Nil, Nil, (BlockLiteral(Nil, vparams, bparams, body)) :: Nil))
+     case Do(effect, id, targs, vargs, bargs) =>
+      Return(Do(effect, id, targs, vargs, (BlockLiteral(Nil, vparams, bparams, body)) :: Nil))
```

Comment:
You are dropping `bargs`, why not (like in the other cases `bargs :+ (BlockLiteral(Nil, vparams, bparams, body))`?

Reply:
Good point!